master/buildbot/www/config.py

```diff
@@ -58,6 +58,8 @@ def renderIndex(self, request):
         config['title'] = self.master.config.title
         config['titleURL'] = self.master.config.titleURL
         config['multiMaster'] = self.master.config.multiMaster
+        if "wamp" in self.master.config.protocols:
+            config['wamp'] = self.master.config.protocols['wamp']
```

Comment:
Why does the web UI need to know if this maser supports slaves connecting via WAMP?